src/cljs/main/broadfcui/endpoints.cljs

```diff
@@ -584,16 +584,3 @@
     :headers ajax/content-type=json
     :data (utils/->json-string tos-url)}
```

Comment:
@MatthewBemis it looks like you added this in https://github.com/broadinstitute/firecloud-ui/pull/1607/files#diff-52b3931bbc7ac93073942c840c1ce432f92749138d26951a07b66a149db876fbR585

Why are you sending `tos-url`? It looks to me that `tos-url` is the cloud function URL (https://github.com/broadinstitute/firecloud-develop/pull/2919/files#diff-c20ec0c146ec8f9eb41e0e3590e6c98358574d92eef4a2a7de783eb0d7c972d8L32)

Reply:
Or maybe `tos-url` just happens to be the same name as what is defined in https://github.com/broadinstitute/firecloud-ui/pull/1610/files#diff-8800b72bac2b6918985e4cb588d743bbad79f91e40491a4eb9c4163ea35260ceL67? I'm unfamiliar with `cljs`, so it's hard for met to know.

Reply:
The `tos-url` being passed into this is coming from here: https://github.com/broadinstitute/firecloud-ui/pull/1607/files#diff-38d6a2a825de69453b37819e66002057e11717a3d981ae55cc06faa712d3792bR334

So yes, your 2nd comment is correct

Reply:
Thanks!